src/Orchard.Web/Core/Contents/Views/Admin/CreatableTypeList.cshtml

```diff
@@ -1,5 +1,26 @@
-@{ Layout.Title = T("Create New Content").ToString(); }
+@using Orchard;
+@using Orchard.ContentManagement;
+@using Orchard.Core.Contents;
+
+@{
+    IOrchardServices _orchardServices;
+    WorkContext.TryResolve<IOrchardServices>(out _orchardServices);
+
+    Layout.Title = T("Create New Content").ToString();
```

Comment:
Can you set `T("Create New Content")` without the `ToString` ? There might be double encoding issues otherwise.

Reply:
Done.